### DISCUSSION TITLE
Shared dimensions and map

### DISCUSSION THREAD

**Discussion by AFoletti** [visualize-admin/visualization-tool#433] [Q&A] [answered]
Dear visualize tool,

I just published a new LINDAS dataset (available on TEST and INT). The cube uses the "cantons" shared dimension and my goal is to use visualize to plot a map of it.
However, when I [load the dataset on Visualize](https://test.visualize.admin.ch/en/browse/dataset/https%3A%2F%2Fenergy.ld.admin.ch%2Fsfoe%2Fbfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen%2F1?previous=%7B%22type%22%3A%22organization%22%2C%22iri%22%3A%22https%3A%2F%2Fregister.ld.admin.ch%2Fopendataswiss%2Forg%2Fbundesamt-fur-energie-bfe%22%2C%22includeDrafts%22%3Afalse%7D) (TEST, see question 2 below) the "Map" chart is grayed out and impossible to select.

So, a couple of questions:
1) what should I do in order to be able to use the "map" option in visualize?
2) how is the LINDAS-Visualize mapping? I have my dataset published on LINDAS TEST and INT, but is only visible on Visualize TEST. Is that intended?

**Comment by bprusinowski** [comment, marked as answer]
Dear @AFoletti,

I took a look at the Cube Creator and it seems that the data kind of `Kanton` dimension is not set to `geographicShape` (so even when it's mapped to a Shared Dimension it won't be treated as geographical dimension).

This can be changed in the third step (Cube Designer) when editing a particular column; afterwards you should see an icon that indicates successful mapping:
<img width="153" alt="Screenshot 2022-02-28 at 10 29 30" src="https://user-images.githubusercontent.com/52032047/155958700-ee1e9dbf-3753-4960-b30b-0f3e6b056ee3.png">

When it comes to your second question, we are showing datasets from LINDAS INT on Visualize TEST and datasets from LINDAS PROD on Visualize INT (& PROD), so if you'd like to see this dataset on both environments, there would be a need to publish it through production environment of the Cube Creator.

 Let me know if that helps!

**Reply by AFoletti** [reply]
Thanks!
